packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,175 @@
+
+To check that the Device Authorization grant type is enabled:
+
+1. In the left navigation pane of the Admin Console, go to **Security** > **API** and select the "default" Custom Authorization Server. Note that the examples in this guide use the "default" Custom Authorization Server.
+1. On the **Access Policies** tab, select the access policy that you want to configure Device Authorization for.
+1. Click the pencil icon for the Default Policy Rule.
+1. In the Edit Rule dialog box, select **Device Authorization** for the grant type and click **Update Rule**.
+
+## Configure the smart device
+
+### Request the device verification code
+
+The smart device first needs to call the `/device/authorize` endpoint to obtain the unique verification code.
+
+**Request example**
+
+```bash
+  curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/device/authorize \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'scope=openid profile offline_access'
+```
+
+The POST request passes the following parameters:
+
+* `client_id`, which matches the Client ID of the OAuth 2.0 application that you created
+* `scope` to specify which access privileges are being requested for access tokens. See [Scopes](https://developer.okta.com/docs/reference/api/oidc/#scopes) for a list of supported scopes.
+
+**Response example**
+
+```json
+{
+   "device_code": "f5ec0430-c2fe-4a93-912d-08631f1effed",
+   "user_code": "HMBSRJRC",
+   "verification_uri": "https://${yourOktaDomain}/activate",
+   "expires_in": 600,
+   "interval": 5
+}
+```
+
+The properties in the response are:
+
+* `device_code`: The long string that the device uses to exchange for an access token.
+* `user_code`: The text that you enter at the URL that is listed as the value for `verification_uri`.
+* `verification_uri`: The URL that the user needs to enter into their phone to start signing in.
+* `expires_in`: The number of seconds that this set of values is valid. After the device code and user code expire, the user has to start the device verification process over.
+* `interval`: The number of seconds that the device should wait between polling to see if the user has finished signing in.
+
+The `user_code` and `verification_uri` must appear on the smart device for the user.
+
+> **Note:** A QR code compliant `verification_uri` is coming soon.
+
+#### Example of the display on the smart device
+
+![Verification on the smart device:](/img/DeviceAuthGrant2.png)
+
+### Request for user access, ID, and refresh tokens
+
+To retrieve tokens for the user, the smart device needs to make a request to the `/token` endpoint.
+
+**Request example**
+
+```bash
+  curl --location --request POST 'https://${yourOktaDomain}/oauth2/default/v1/token' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'scope=openid profile' \
```

Comment:
It's confusing to request scopes in this request as you already did in `/device/authorize` request. I suggest to remove this line.

Reply:
Removed.